apps/src/p5lab/P5Lab.js

```diff
@@ -1151,7 +1151,7 @@ P5Lab.prototype.initInterpreter = function(attachDebugger = true) {
   this.JSInterpreter.parse({
     code,
     projectLibraries: this.level.projectLibraries,
-    blocks: dropletConfig.blocks,
+    blocks: this.isSpritelab ? [] : dropletConfig.blocks,
```

Comment:
after reading through the slack thread you linked, i get the feeling that this change will have a relatively low risk of regression. my thinking is: spritelab doesn't have text mode, so the likelihood that a user would somehow add a droplet block in their project is really low. is that the right way to think about this? other thoughts on the possibility of regression here?

Reply:
Good question! I'm reasonably confident there's no way to get droplet code into a spritelab project (and even if someone somehow did, I think we'd be okay with this change breaking their project).

The other regression potential to be concerned about is if there are blocks other than `playSound` that were relying on the droplet config. To double check that, I pulled a list of all the spritelab blocks, and removed all the ones that are defined either in the levelbuilder library or in spritelab commands. That left a pretty short list that was pretty easy to check through manually to confirm that none of them rely on Gamelab's droplet config:
```
>>> blocks = set(['setTint', 'mirrorSprite', 'repeatForever', 'drawLine', 'followingTargets', 'pointInDirection', 'checkTouching', 'startGliding', 'getTime', 'tumbling', 'isTouchingSprite', 'removeTint', 'bounceOff', 'setBackgroundImage', 'setSizes', 'stopCollectingData', 'startFollowing', 'destroy', 'makeNewSpriteAnon', 'moveBackward', 'removeAllBehaviors', 'whenTouching', 'moveToward', 'countByAnimation', 'textVariableJoin', 'whileUpArrow', 'spriteSay', 'beginCollectingData', 'removeBehaviorSimple', 'withPercentChance', 'ifVarEquals', 'atTime', 'isTouchingEdges', 'createNewSprite', 'yLocationOf', 'textJoin', 'isCostumeEqual', 'whenPromptAnswered', 'locationMouse', 'whenAllPromptsAnswered', 'setProp_dropdown', 'makeNumSprites', 'clickedSpritePointer', 'moveInDirection', 'keyPressed', 'jumpTo', 'spriteSayChoices', 'setBackgroundImageAs', 'updateTitleWithColor', 'mouseLocation', 'locationAt', 'whileRightArrow', 'withPercentChanceDropdown', 'patrollingUpDown', 'turn', 'comment', 'whenRightArrow', 'spriteClicked', 'setupScoreboard', 'spriteSayTime', 'getThisSprite', 'avoidingTargets', 'addTarget', 'whileLeftArrow', 'hideTitleScreen', 'randColor', 'edgesDisplace', 'whileTouching', 'setBackground', 'isKeyPressed', 'location_picker', 'stopFollowing', 'whenDownArrow', 'xLocationOf', 'subjectSpritePointer', 'whenSpriteCreated', 'randomLocation', 'drawLine.js', 'playSound', 'objectSpritePointer', 'changePropBy', 'stopAvoiding', 'locationModifier', 'setPrompt', 'moveForward', 'addBehaviorSimple', 'setProp', 'setAnimation', 'locationOf', 'getAllSprites', 'setPromptWithChoices', 'printText', 'whenLeftArrow', 'showTitleScreen', 'randomColor', 'whileKey', 'whenKey', 'startAvoiding', 'stopGliding', 'clickedOn', 'whileDownArrow', 'allSpritesWithAnimation', 'glideTo', 'getProp', 'newSpritePointer', 'draggable', 'whenUpArrow', 'setDefaultSpriteSize', 'setupSim', 'costumeImage'])
>>> libraryFuncs = set(['whenRightArrow', 'whileTouching', 'yLocationOf', 'followingTargets', 'withPercentChance', 'avoidingTargets', 'pointInDirection', 'randColor', 'whileDownArrow', 'whenLeftArrow', 'startGliding', 'whileLeftArrow', 'tumbling', 'randomColor', 'whileKey', 'whenDownArrow', 'whileUpArrow', 'startAvoiding', 'wordWrap', 'draw', 'stopFollowing', 'setSizes', 'xLocationOf', 'stopGliding', 'withPercentChanceDropdown', 'startFollowing', 'spriteSayChoices', 'clickedOn', 'mouseLocation', 'whenKey', 'whileRightArrow', 'patrollingUpDown', 'ifVarEquals', 'spriteSayTime', 'whenTouching', 'draggable', 'speechBubbles', 'whenUpArrow', 'spriteSay', 'beginCollectingData'])
>>> commands = set(['comment', 'setTint', 'mirrorSprite', 'repeatForever', 'addBehaviorSimple', 'drawTitle', 'getTime', 'removeAllBehaviors', 'atTime', 'getAnimationsInUse', 'isTouchingEdges', 'collectData', 'createNewSprite', 'setPrompt', 'getPrintLog', 'whenAllPromptsAnswered', 'getBackground', 'getThisSprite', 'textVariableJoin', 'isCostumeEqual', 'removeBehaviorSimple', 'addTarget', 'getTitle', 'checkTouching', 'whenPromptAnswered', 'hideTitleScreen', 'locationMouse', 'bounceOff', 'turn', 'makeNewSpriteAnon', 'keyPressed', 'edgesDisplace', 'moveInDirection', 'setBackground', 'showTitleScreen', 'setBackgroundImage', 'destroy', 'makeNumSprites', 'isTouchingSprite', 'spriteClicked', 'locationOf', 'removeTint', 'draggableFunc', 'executeDrawLoopAndCallbacks', 'getEventLog', 'displace', 'stopCollectingData', 'glideFunc', 'whenSpriteCreated', 'randomLocation', 'Behavior', 'isKeyPressed', 'playSound', 'followingTargetsFunc', 'locationAt', 'setBackgroundImageAs', 'glideTo', 'getProp', 'changePropBy', 'jumpTo', 'moveBackward', 'locationModifier', 'getNumBehaviorsForAnimation', 'avoidingTargetsFunc', 'getBehaviorsForSpriteId', 'setProp', 'spriteSay', 'getNumBehaviorsForSpriteId', 'moveToward', 'setAnimation', 'printText', 'countByAnimation', 'getPromptVars', 'setDefaultSpriteSize', 'setPromptWithChoices', 'moveForward', 'getSpriteIdsInUse', 'textJoin'])
>>> blocks = blocks - libraryFuncs
>>> blocks = blocks - commands
>>> blocks
set([
'location_picker', # simple value
'newSpritePointer', # simple value
'stopAvoiding', # broken (probably never worked)
'drawLine', # block helper code
'subjectSpritePointer', # simple value
'getAllSprites', # expression
'updateTitleWithColor', # broken (probably never worked)
'objectSpritePointer', # simple value
'setupScoreboard', # setup sim library
'costumeImage', # expression
'drawLine.js', # block helper code 
'setProp_dropdown', # broken (probably never worked)
'setupSim', # setup sim library
'allSpritesWithAnimation', # expression
'clickedSpritePointer' # simple value
])

Reply:
very cool -- i wouldn't have thought to test that way!